modules/core/src/test/java/org/apache/ignite/internal/binary/BinaryObjectBuilderAdditionalSelfTest.java

```diff
@@ -1974,6 +1974,40 @@ public void testBuilderReusage() throws Exception {
         assertEquals("val2", builder.build().field("f1"));
     }
 
+    /**
+     * @throws Exception If fails
+     */
+    @Test
+    public void testGetFieldForNotAssignedValInEmptyBuilder() {
```

Comment:
Please fix style: abbreviation shouldn't be used at method name.
suggest: `testGetFieldForNotAssignedValInEmptyBuilder` -> `testGetFieldForNotAssignedValueInEmptyBuilder` or  `testGetNotAssignedFieldInEmptyBuilder`

Reply:
Fixed